CMakeLists.txt

```diff
@@ -684,6 +680,19 @@ if(NOT IOS AND NOT XROS)
   include(cmake/OpenCVDetectPython.cmake)
 endif()
 
+# Use same flags for native AArch64 and RISC-V compilation as for cross-compile (Linux)
+if((CV_GCC OR CV_CLANG) AND UNIX AND NOT APPLE AND NOT CMAKE_CROSSCOMPILING AND NOT CMAKE_TOOLCHAIN_FILE)
```

Comment:
Lets move out that from the root CMake file.

Reply:
Moved to `cmake/platforms/OpenCV-Linux.cmake`